modules/cij/runner.py

```diff
@@ -286,7 +288,12 @@ class TestRun(Runnable):
     # pylint: disable=too-many-instance-attributes
 
     ver: str = ""
-    conf: dict = dataclasses.field(default_factory=dict)
+    args: argparse.Namespace = dataclasses.field(
+        default_factory=argparse.Namespace
+    )
+    conf: cij.conf.Config = dataclasses.field(
```

Comment:
These type annotations are great -- we should do the same for some of the other fields, e.g. testcases and testsuites. I'll put that on my TODO